src/HotChocolate/AspNetCore/src/AspNetCore.Playground/PlaygroundOptions.cs

```diff
@@ -17,6 +18,27 @@ public PlaygroundOptions()
             _path = _defaultPath = new PathString("/playground");
         }
 
+        /// <summary>
+        /// The endpoint of the GraphQL service.
+        /// </summary>
+        public string? GraphQLEndpoint
```

Comment:
Should we even make this a URI so there cannot be an exception? @PascalSenn thoughts?

Reply:
I generally do not like overring getter and setter. 
So asking me, yes we should

Reply:
Ha, same here! Was aiming to be consistent with the rest of the class 😄. Will update.

Reply:
So lets make the property a Uri?